modules/samples-operator-overview.adoc

```diff
@@ -7,37 +7,37 @@
-`openshift-cluster-samples-operator` namespace. On startup, it copies the pull
-secret captured by the installation into the OpenShift namespace with the name
-`samples-registry-credentials` to facilitate image stream imports. A cluster
-administrator can create any additional secrets containing the content of a
-Docker `config.json` in the OpenShift namespace as needed to facilitate image
-import.
+On initial startup, the operator creates the default config object for itself
+and then creates the sample image streams and templates. The image streams are
+RHEL {product-title} based and point to images on registry.redhat.io. The
+templates are those categorized as {product-title} Templates. It also copies the
+pull secret captured by the installation into the OpenShift namespace with the
+name `samples-registry-credentials` to facilitate image stream imports. A
+cluster administrator can create any additional secrets containing the content
+of a Docker `config.json` in the OpenShift namespace as needed to facilitate
+image import.
+
+The Samples Operator configuration is a cluser-wide resource, and the deployment
+is contained within the `openshift-cluster-samples-operator` namespace.
 
 The image for the Samples Operator contains image stream and template
-definitions for the associated OpenShift release. Each sample includes an
-annotation that denotes the OpenShift version with which it is compatible. The
-operator uses this annotation to ensure that each sample matches the release
-version. Samples outside of its inventory are ignored, as are skipped samples.
-Modifications to any samples that are managed by the operator will be reverted
-automatically. The Jenkins images are actually part of the image payload from
-installation and are tagged into the image streams directly.
-
-The samples resource includes a finalizer which cleans up the following upon
-deletion:
-
-* Operator managed image streams
-* Operator managed templates
-* Operator generated configuration resources
-* Cluster status resources
-* The `samples-registry-credentials` secret
+definitions for the associated OpenShift release. When each sample is created or
+updated, the Samples Operator includes an annotation that denotes the
+version of {product-title}. The operator uses this annotation to ensure
+that each sample matches the release version. Samples outside of its inventory
+are ignored, as are skipped samples. Modifications to any samples that are
+managed by the operator will be reverted automatically. The Jenkins images are
+actually part of the image payload from installation and are tagged into the
+image streams directly.
+
+The Samples Operator configuration resource includes a finalizer which cleans up
+the following upon deletion:
+
+* Operator managed imagestreams.
+* Operator managed templates.
+* Operator generated configuration resources.
+* Cluster status resources.
+* The `samples-registry-credentials` secret.
```

Comment:
we are planning to no longer delete the secret with https://github.com/openshift/cluster-samples-operator/pull/125 ... please track that PR, and assuming that goes through, remove this line please